hcsshim.go

```diff
@@ -119,3 +162,17 @@ func convertAndFreeCoTaskMemString(buffer *uint16) string {
 	coTaskMemFree(unsafe.Pointer(buffer))
 	return str
 }
+
+func convertAndFreeCoTaskMemBytes(buffer *uint16) ([]byte, error) {
+	// TODO this requires an extra copy of the returned string data,
+	// it should be replaced with a more efficient way.
+	return []byte(convertAndFreeCoTaskMemString(buffer)), nil
```

Comment:
I'm attempting to figure out why my method to do that was not working. I pushed this as a temporary hack.